platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java

```diff
@@ -53,6 +49,7 @@
  */
 public class ExperimentManager implements Closeable {
     private static final Logger LOGGER = LoggerFactory.getLogger(ExperimentManager.class);
+    private static final int MAX_EXECUTION_TIME = 20 * 60 * 1000;
```

Comment:
Having a default max execution time that can be sued as fallback is very good. However, the name is missleading. You should swap it with the `DEFAULT_MAX_EXECUTION_TIME` from line 69.